devices/AtomQrCode/ScanTimeout.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.AtomQrCode
+{
+    /// <summary>
+    /// Values for scan timeout.
+    /// </summary>
+    public enum ScanTimeout
+    {
+        /// <summary>
+        /// Timeout 1 second.
+        /// </summary>
+        Timeout_1sec = 1,
```

Comment:
minim but there should not be underscore.

Reply:
To follow the guidelines it should start with `Timeout` either... 😉